tests/unit/Endpoints/Webhooks/UpdateOrderTest.php

```diff
@@ -85,6 +86,90 @@ public function testOrderOnlySyncMode()
         $this->assertTaskNotCount(0, 'There are suppose to be any tasks');
     }
 
+    public function dataProviderOrderRefund(): array
+    {
+        $data = [];
+
+        $data['backoffice order without refund'] = [
+          'toBePaidBackValueWt' => 50.00,
+          'refundedPriceWt' => 0,
```

Comment:
```suggestion
          'toBePaidBackValueWt' => 50.00,
          'refundedPriceWt' => 50.00,
```

Normally it's like this, first the credit invoice it made and than the money paid back

Reply:
Changed this.
See https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/pull/71#discussion_r916893718